test/profiler-mocha.js

```diff
@@ -172,6 +172,11 @@ describe("profiler", function() {
                nodeInstance.profiler.stopEvent.should.be.a('function');
                nodeInstance.profiler.startUpdate.should.be.a('function');
                nodeInstance.profiler.stopUpdate.should.be.a('function');
+               function delay(ms) {
+                  ms += new Date().getTime();
+                  while (new Date() < ms){}
```

Comment:
This will always return fail even if the clock has not advanced. This may not be noticeable on unix, but I know on Windows the JVM's millisecond clock (as a comparison) is not as accurate as on unix.